application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -2149,6 +2149,8 @@ type Stop implements Node & PlaceInterface {
   geometries: StopGeometries
   "ÌD of the stop in format `FeedId:StopId`"
   gtfsId: String!
+  "Stop has services running in the future"
+  hasFutureServices: Boolean
```

Comment:
I think rather than booleans we want an enum or something even more descriptive.

Reply:
Can the definition of "future" be limited to beyond the timespan of real-time updates here, in order to avoid a real-time update using a normally unused stop returning incorrect values?